internal/translation/deployment/deployment.go

```diff
@@ -57,15 +57,19 @@ func (ds *ProductionAtlasDeployments) ListClusterNames(ctx context.Context, proj
 	return deploymentNames, nil
 }
 
-func (ds *ProductionAtlasDeployments) ListDeploymentConnections(ctx context.Context, projectID string) ([]Connection, error) {
+func (ds *ProductionAtlasDeployments) ListDeploymentConnections(ctx context.Context, projectID string, isGov bool) ([]Connection, error) {
 	clusters, _, err := ds.clustersAPI.ListClusters(ctx, projectID).Execute()
 	if err != nil {
 		return nil, fmt.Errorf("failed to list clusters for project %s: %w", projectID, err)
 	}
 	clusterConns := clustersToConnections(clusters.GetResults())
 
-	serverless, _, err := ds.serverlessAPI.ListServerlessInstances(ctx, projectID).Execute()
-	if err != nil {
+	if isGov {
```

Comment:
A field here would be better:
```suggestion
	if ds.isGov {
```